src/components/CustomTable/StyledTableBody/StyledTableBody.js

```diff
@@ -25,7 +27,12 @@ const StyledTableBody = ({ columns, data }) => {
           hover
           key={row.id}
           classes={{ root: classes.root, hover: classes.hover }}
-          onClick={() => alert("You have clicked row " + row.name)}
+          onClick={() =>
+            router.push({
+              pathname: "/characters",
+              query: { params: ["edit", row.id] },
+            })
+          }
```

Comment:
This is supposed to be a reusable component, what would happen if this component was to be used elsewhere?